src/Entity/Version.php

```diff
@@ -710,6 +710,41 @@ public function setPhpExt(?array $phpExt): void
         $this->phpExt = $phpExt;
     }
 
+    public function getPieName(): ?string
+    {
+        if (!$this->isPiePackage()) {
+            return null;
+        }
+        $name = $this->getPhpExt()['extension-name'] ?? explode('/', $this->getName())[1];
+
+        if (!str_starts_with($name, 'ext-')) {
+            $name = 'ext-'.$name;
+        }
+
+        return $name;
+    }
+
+    /**
+     * @return list<string>
+     */
+    public function getPieSupportedOS(): array
+    {
+        if (!$this->isPiePackage()) {
+            return [];
+        }
+
+        if (isset($this->phpExt['os-families'])) {
+            return $this->phpExt['os-families'];
+        }
+
+        return array_values(array_diff(["windows", "bsd", "darwin", "solaris", "linux", "unknown"], $this->phpExt['os-families-exclude'] ?? ['any']));
```

Comment:
shouldn't this use `?? []` for the case of no exclusions ? I would make more sense than using `any` as a string that is not one of the supported families.